defaults/main.yml

```diff
@@ -8,15 +8,16 @@ lin_rclone_repo_owner: rclone
 lin_rclone_binary: /usr/bin/rclone
 lin_rclone_config: /etc/rclone/rclone.conf
 
-lin_rclone_allow_nonroot: no
+lin_rclone_allow_nonroot: yes
 lin_rclone_group_gid: 911
 
 lin_rclone_idle_timeout: 900
 lin_rclone_mount_timeout: 10
-lin_rclone_vfs_cache_mode: writes
-lin_rclone_dir_cache_time: 30s
-
+lin_rclone_vfs_cache_mode: minimal
```

Comment:
I based this default on rclone forum observations. It is full of problem reports where people are advised to use `at least writes`.